src/modules/reservations.js

```diff
@@ -0,0 +1,45 @@
+import { shows } from '../utils';
```

Comment:
Optional

- [ ] eslint showed me the warning that this should have the file extension (.js)